### PR TITLE
feat(styles): add css bundles into package

### LINKED ISSUE #11489: Add prebuilt CSS to `@carbon/styles`
It would be helpful if the `@carbon/styles` package had a `css` folder with prebuilt styles. Specifically:

- `styles/css/styles.css`
- `styles/css/styles.min.css` (minified)

This ticket is to introduce another step to the build task in `@carbon/styles` to generate a bundle from the root `index.scss` file.

### PR DESCRIPTION
**Pull request by joshblack**
Closes #11489

This PR adds in a `build` task which generates prebuilt CSS files in the `styles` package.

#### Changelog

**New**

**Changed**

- Update import ordering of `index.scss`
- Add `build.js` task and script in `package.json`

**Removed**

#### Testing / Reviewing

- Pull down the PR
- Run `yarn build` in `packages/styles`
- Verify that `css/styles.css` and `css/styles.min.css` have been generated
- Inspect their contents to make sure that styles are actually being generated 😅 
- Verify that `css` is included in the `files` array of `package.json`

### CODE REVIEW

File: packages/styles/tasks/build-css.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+const postcss = require('postcss');
+const sass = require('sass');
+const autoprefixer = require('autoprefixer');
+const cssnano = require('cssnano');
+const flexbugs = require('postcss-flexbugs-fixes');
+
+async function build() {
+  const filepath = path.resolve(__dirname, '..', 'index.scss');
+  const loadPaths = [
+    path.resolve(__dirname),
```
Comment (joshblack) [inline packages/styles/tasks/build-css.js]:
```suggestion
    __dirname,
```

**Comment by theiliad** [comment]
🎉 🎉 🎉

**Comment by theiliad** [comment]
Thanks @joshblack for adding this!

**Review by theiliad — APPROVED** [review]

**Review by aledavila — APPROVED** [review]
works as expected

**Review by dakahn — APPROVED** [review]
worked as expected 🏄🏾

**Pull request merged**